Mobile_version.css

```diff
@@ -41,6 +41,52 @@ ul {
   text-decoration: none;
 }
 
+.Popup {
+  height: 5130px;
+  width: 375px;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+  position: fixed;
+  background-color: #6070ff;
+  z-index: 1;
+  display: none;
+}
```

Comment:
**OPTIONAL** - When I switch from the mobile version to the desktop version, the mobile menu is still very visible, this is not the expected behavior of the application.Please fix this 🙏 

Your project:
<img width="948" alt="tctdtd" src="https://user-images.githubusercontent.com/73607512/210354777-146a7064-81ac-49b5-8b76-981f91dcaafb.PNG">